tests/robot-tests/tests/admin_and_public_2/bau/publish_release_and_amend_2.robot

```diff
@@ -253,7 +253,7 @@ Create release amendment
 Replace subject data
     user uploads subject replacement    ${SUBJECT_NAME}    dates.csv    dates.meta.csv
     user waits until page contains element    testid:Data file replacements table
-    user confirms replacement upload    ${SUBJECT_NAME}
+    user confirms replacement upload    ${SUBJECT_NAME}    Error
```

Comment:
I'm not sure that I follow why this data set needs altering.

I read your earlier explanation, but I don't think this data set has a related Public API data set?

Reply:
Cool, thanks, I didn't realise it displays as 'Error' when additional action is required for data blocks and footnotes as well as the Public API.

I think this is fine then, but as suggested earlier, I think we should discuss changing this label to something like 'Action required' rather than 'Error'.